src/config.ts

```diff
@@ -49,9 +49,15 @@ export interface SwaggerConfig {
 
   /**
    * API host, expressTemplat e.g. localhost:3000 or myapi.com, use null for relative urls
+   * OAPI2 compliant
    */
   host?: string | null;
 
+  /**
+   * API hosts array, e.g. [https://locahost:3000, https://myapi.com]
+   */
+  hosts?: string[];
```

Comment:
```suggestion
  servers?: OpenAPI3Server[];
```
See https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#serverObject